styles.css

```diff
@@ -163,3 +168,53 @@ footer a:hover {
 }
 
 /* Media queries */
+
+
+@media (min-width: 768px) and (max-width: 1279px) {
+  .splash {
+    display: none;
+  }
+
+main{
+  width: 677.6875px;
+}
+
+footer {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+}
+}
+
+@media (min-width: 1280px){
+  main {
+    width: 885.5px;
+  }
+}
+
+@media (max-width: 768px){
```

Comment:
You already have a media query that takes care of the possibility that your screen is 768 pixels. How can you update this  max-width so that it doesn't change the styling you already set for 768 pixels?